### PR TITLE
PIE-1718/REST API endpoint documentation for Run#show

### PR DESCRIPTION
**Pull request by amybiyuliu**
**Description**

Documentation about Test Analytics rest api endpoint Run#show

Linear: https://linear.app/buildkite/issue/PIE-1718/rest-api-endpoints-documentation-runshow
Screenshot 2023-06-07 at 4 28 43 pm

### CODE REVIEW

**Review by gchan — APPROVED** [review]
LGTM!

**Comment by gchan** [comment]
I removed the 'result' attribute from the sample response (see discussion https://github.com/buildkite/buildkite/pull/12500#discussion_r1223733986). Can this get another set of eyes for review? cc @swebb @mbelton-buildkite

**Review by swebb — APPROVED** [review]
LGTM 👍🏻

File: pages/apis/rest_api/analytics/runs.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+```json
+
+  {
+    "id": "64374307-12ab-4b13-a3f3-6a408f644ea2",
+    "branch": "main",
+    "commit_sha": "e32432c045439c95b5af2cb472b4d8be2207685d",
+    "url": "https://api.buildkite.com/v2/analytics/organizations/my_great_org/suites/my_suite_slug/runs/64374307-12ab-4b13-a3f3-6a408f644ea2",
+    "web_url":"https://buildkite.com/organizations/my_great_org/analytics/suites/my_suite_slug/runs/64374307-12ab-4b13-a3f3-6a408f644ea2"
+  }
+
```
Comment (mbelton-buildkite) [inline pages/apis/rest_api/analytics/runs.md:18]: Looks like there's some extra spacing here:

```suggestion
{
  "id": "64374307-12ab-4b13-a3f3-6a408f644ea2",
  "result": "passed",
  "branch": "main",
  "commit_sha": "e32432c045439c95b5af2cb472b4d8be2207685d",
  "url": "https://api.buildkite.com/v2/analytics/organizations/my_great_org/suites/my_suite_slug/runs/64374307-12ab-4b13-a3f3-6a408f644ea2",
  "web_url":"https://buildkite.com/organizations/my_great_org/analytics/suites/my_suite_slug/runs/64374307-12ab-4b13-a3f3-6a408f644ea2"
}
```

**Review by mbelton-buildkite — APPROVED** [review]
Just a spacing/alignment issue on the example response, otherwise looks good 😊

File: pages/apis/rest_api/analytics/runs.md (hunk 1 above)
Comment (gchan) [reply pages/apis/rest_api/analytics/runs.md:18]: Good catch! We will need to go back and fix a similar spacing issue on the suites page.

Comment (gchan) [reply pages/apis/rest_api/analytics/runs.md]: I've amended the commit!

File: pages/apis/rest_api/analytics/runs.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Runs API
+
+## Get a run
+
+```bash
+curl "https://api.buildkite.com/v2/analytics/organizations/{org.slug}/suites/{suite.slug}/runs/{run.id}"
+```
+
+```json
+
```
Comment (swebb) [inline pages/apis/rest_api/analytics/runs.md:10]: [nit] Unnecessary white space?

**Pull request merged**